### ISSUE TITLE
Kestrel vs Beta8

### ISSUE THREAD

**Issue by rhires** [aspnet/Home#1002]
dnx kestrel throws an exception when running beta 8, and my project.json showing beta8 `"Microsoft.AspNet.Server.Kestrel": "1.0.0-beta8",` - notably, it will run if I have beta7 in there for Kestrel, even though dnx is beta8. Running mono on Mac OS X Yosemite. I recall seeing something about the commands section being different for the kestrel commands, as a possible solution, but I can't seem to find it. Anyway, here's the first part of the exception:
`System.FormatException: Could not parse the JSON file. Error on line number '0': 'server.urls=http://localhost:5000'. ---> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: s. Path '', line 0, position 0.` I can post the rest if anyone is interested.

**Comment by davidfowl** [comment]
Publish your project to a github repo

**Comment by rhires** [comment]
Here's the repo: https://github.com/rhires/kestrelvsbeta8. I realize that I didn't previously mention that I used yeoman to scaffold the thing, then went in and changed the dependencies to beta8 from beta7, and ran a dnu restore...

**Comment by Tratcher** [comment]
hosting.ini has changed to hosting.json.

**Comment by rhires** [comment]
Cool. What would be the proper format for the info already in that file? I guessed `	"server.urls":"http://localhost:5000"` and it seems to be right, but when I updated yeoman, and it scaffolded out a new project, it doesn't bother with hosting.anything, and instead it seems that the server.urls is hard coded somewhere, because I can't see where it would be configured (I looked in all the different files that the scaffold provides...), and yet it still opens port 5000. As I stray off topic a bit (because yeoman is suggested by, but isn't controlled by, the aspnet folks), I also notice that the yeoman beta8 scaffold merely includes "web" as its command to activate kestrel, and no configuration. 

Anyway, I'm able to go my merry way since I can run kestrel again. :-) Thanks!
